dir.props

```diff
@@ -94,6 +96,11 @@
     <ReferencePackagesSourceDir>$(ReferencePackagesBaseDir)source/</ReferencePackagesSourceDir>
     <ReferencePackagesDir>$(ReferencePackagesBaseDir)packages/</ReferencePackagesDir>
     <ReferencePackagesToDeleteDir>$(ReferencePackagesBaseDir)packages-to-delete/</ReferencePackagesToDeleteDir>
+    <BaselineDataFile>$(ToolsLocalDir)prebuilt-baseline-</BaselineDataFile>
+    <OfflineBaselineDataFile>$(BaselineDataFile)offline.xml</OfflineBaselineDataFile>
+    <OnlineBaselineDataFile>$(BaselineDataFile)online.xml</OnlineBaselineDataFile>
+    <BaselineDataFile Condition="'$(OfflineBuild)' == 'true'">$(OfflineBaselineDataFile)</BaselineDataFile>
+    <BaselineDataFile Condition="'$(OfflineBuild)' != 'true'">$(OnlineBaselineDataFile)</BaselineDataFile>
```

Comment:
Any plans for these conditional sets? (I think they should be removed otherwise.)

Reply:
Yes.  They're being used to identify the online and offline baseline file to pass in to the WriteUsageBurndownData task.